datasets/wikitablequestions/wikitablequestions.py

```diff
@@ -0,0 +1,183 @@
+This WikiTableQuestions dataset is a large-scale dataset for the task of question answering on semi-structured tables.
+"""
+
+_HOMEPAGE = "https://nlp.stanford.edu/software/sempre/wikitable"
+
+_LICENSE = "Creative Commons Attribution Share Alike 4.0 International"
+
+# The HuggingFace Datasets library doesn't host the datasets but only points to the original files.
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_DATA_URL = (
+    "https://github.com/ppasupat/WikiTableQuestions/releases/download/v1.0.2/WikiTableQuestions-1.0.2-compact.zip"
+)
+
+
+class WikiTableQuestions(datasets.GeneratorBasedBuilder):
+    """WikiTableQuestions: a large-scale dataset for the task of question answering on semi-structured tables."""
+
+    VERSION = datasets.Version("1.0.2")
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('my_dataset', 'first_domain')
+    # data = datasets.load_dataset('my_dataset', 'second_domain')
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="random-split-1",
+            version=VERSION,
+            description="The random-split-1-train/dev.tsv and pristine-unseen-tables.tsv",
+        ),
+        datasets.BuilderConfig(
+            name="random-split-2",
+            version=VERSION,
+            description="The random-split-2-train/dev.tsv and pristine-unseen-tables.tsv",
+        ),
+        datasets.BuilderConfig(
+            name="random-split-3",
+            version=VERSION,
+            description="The random-split-3-train/dev.tsv and pristine-unseen-tables.tsv",
+        ),
+        datasets.BuilderConfig(
+            name="random-split-4",
+            version=VERSION,
+            description="The random-split-4-train/dev.tsv and pristine-unseen-tables.tsv",
+        ),
+        datasets.BuilderConfig(
+            name="random-split-5",
+            version=VERSION,
+            description="The random-split-5-train/dev.tsv and pristine-unseen-tables.tsv",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = (
+        "random-split-1"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+    )
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "id": datasets.Value("string"),
+                "question": datasets.Value("string"),
+                "answer": datasets.features.Sequence(datasets.Value("string")),
```

Comment:
Maybe rename this column "answers" with an 's', since there can be several of them ?

Reply:
Agree with that. I will rename that.

Reply:
Rename it in https://github.com/huggingface/datasets/pull/3870/commits/7ae6228531ec7ce7e495bff66f8b47c2be678bba

Reply:
Fix remaining inconsistency in https://github.com/huggingface/datasets/pull/3870/commits/143474b494e0220080f6b687c73425f127b64b17